packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -388,6 +389,43 @@ class RelativePositionedTransition extends AnimatedWidget {
   }
 }
 
+/// Animated version of a [DecoratedBox] that animates the different properties
+/// of its [Decoration].
```

Comment:
You might also want to cross link to `AnimatedContainer` (i.e., have "See also" bullets that go both ways).